docs/source/en/package_reference/inference_endpoints.md

```diff
@@ -0,0 +1,46 @@
+# Inference Endpoints
+
+Inference Endpoints offers a secure production solution to easily deploy any `transformers`, `sentence-transformers` and `diffuser` models from the Hub on a dedicated and autoscaling infrastructure managed by Hugging Face.
+
+A Hugging Face Endpoint is built from a [Hugging Face Model Repository](https://huggingface.co/models). When an Endpoint is created, the service creates image artifacts that are either built from the model you select or a custom-provided container image. The image artifacts are completely decoupled from the Hugging Face Hub source repositories to ensure the highest security and reliability levels.
+
+🤗 Inference Endpoints support all of the `transformers`, `sentence-transformers` and `diffuser` tasks as well as custom tasks not supported by `transformers` yet like speaker diarization and diffusion.
```

Comment:
Maybe remove diffusion as an example since its supported by the 🤗 Diffusers library and not 🤗 Transformers. Super minor nit, but also standardizing whether we're using an emoji with Inference Endpoints. I'm happy with either way, but since Inference Endpoints was introduced without an emoji, I'll stick with that convention here :)

```suggestion
Inference Endpoints support all of the 🤗 Transformers, sentence-transformers, and 🤗 Diffusers tasks, as well as custom tasks not supported by 🤗 Transformers yet, like speaker diarization.
```

Reply:
I'll remove the :hugs:, thanks for raising the question :)

```suggestion
Inference Endpoints support all of the `transformers`, `sentence-transformers` and `diffusers` tasks as well as custom tasks not supported by `transformers` yet like speaker diarization.
```

Reply:
yes let's go easy on the emojis...